railties/test/commands/log_test.rb

```diff
@@ -0,0 +1,93 @@
+# frozen_string_literal: true
+
+require "isolation/abstract_unit"
+require "rails/command"
+require "rails/commands/log/log_command"
+
+class Rails::Command::LogTest < ActiveSupport::TestCase
+  include ActiveSupport::Testing::Isolation
+  setup :build_app
+  teardown :teardown_app
+
+  test "`rails log:clear` clears all environments log files by default" do
+    Dir.chdir(app_path) do
+      File.open("config/environments/staging.rb", "w")
```

Comment:
I realize this is just copied from the original test, but this opens a file and leaves the file open until it is collected by GC.  We could use `File.write` like the lines further below, or we could use the `app_file` helper:

```suggestion
      app_file "config/environments/staging.rb", ""
```

If we change all the lines to use the `app_file` and `app_path(_)` helpers, we can drop the `Dir.chdir` block.

Reply:
I've refactored the tests to use `app_file`.